packages/types/src/common/config-module.ts

```diff
@@ -342,6 +342,8 @@ export type ProjectConfigOptions = {
    *   // ...
    * }
    * ```
+   *
+   * @deprecated use `database_driver_options` instead
```

Comment:
question: For Typeorm connections, should this be backward compatible?

Reply:
The typeorm datasource is not loaded if the v2 flag is enabled

Reply:
My thinking was that if we release this tomorrow, `database_extra` will be denoted as deprecated, and v1 users will attempt to use `database_driver_options` instead, which is incompatible with the Typeorm datasource. I might be overthinking here, let me know if it makes sense.

Reply:
think it's fine to leave out the deprecation warning for now - when we get to v2 we should probably remove it altogether

Reply:
ahah you are right sorry, I ll remove that flag